package.json

```diff
@@ -95,6 +95,7 @@
         "eslint-plugin-import": "2.25.3",
         "eslint-plugin-jsdoc": "37.1.0",
         "eslint-plugin-jsx-a11y": "6.5.1",
+        "eslint-plugin-mui-path-imports": "0.0.15",
```

Comment:
Why do we need this?

Reply:
Yes. Due to adding that plugin to our eslint config package